lib/modules/platform/gerrit/index.ts

```diff
@@ -77,10 +81,32 @@ export function initPlatform({
   config.gerritUsername = username;
   defaults.endpoint = ensureTrailingSlash(endpoint);
   setBaseUrl(defaults.endpoint);
+
+  let gerritVersion: string;
+  try {
+    const env = getEnv();
+    /* v8 ignore start: experimental feature */
```

Comment:
start-stop comments no longer work 
```suggestion
    /* v8 ignore if: experimental feature */
```